client/modules/IDE/components/Editor/index.jsx

```diff
@@ -1,61 +1,82 @@
+  keymap,
+  highlightActiveLine,
+  lineNumbers,
+  Decoration
+} from '@codemirror/view';
 import Fuse from 'fuse.js';
-import emmet from '@emmetio/codemirror-plugin';
+import { javascript } from '@codemirror/lang-javascript';
+import { css } from '@codemirror/lang-css';
+import {
+  bracketMatching,
+  foldGutter,
+  syntaxHighlighting,
+  codeFolding
+} from '@codemirror/language';
+import { autocompletion, closeBrackets } from '@codemirror/autocomplete';
+import { linter, lintGutter } from '@codemirror/lint';
+import { standardKeymap } from '@codemirror/commands';
+import MediaQuery from 'react-responsive';
 import prettier from 'prettier/standalone';
 import babelParser from 'prettier/parser-babel';
 import htmlParser from 'prettier/parser-html';
 import cssParser from 'prettier/parser-postcss';
+import { debounce } from 'lodash';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import { withTranslation } from 'react-i18next';
-import StackTrace from 'stacktrace-js';
-import 'codemirror/mode/css/css';
-import 'codemirror/mode/clike/clike';
-import 'codemirror/addon/selection/active-line';
-import 'codemirror/addon/lint/lint';
-import 'codemirror/addon/lint/javascript-lint';
-import 'codemirror/addon/lint/css-lint';
-import 'codemirror/addon/lint/html-lint';
-import 'codemirror/addon/fold/brace-fold';
-import 'codemirror/addon/fold/comment-fold';
-import 'codemirror/addon/fold/foldcode';
-import 'codemirror/addon/fold/foldgutter';
-import 'codemirror/addon/fold/indent-fold';
-import 'codemirror/addon/fold/xml-fold';
-import 'codemirror/addon/comment/comment';
-import 'codemirror/keymap/sublime';
-import 'codemirror/addon/search/searchcursor';
-import 'codemirror/addon/search/matchesonscrollbar';
-import 'codemirror/addon/search/match-highlighter';
-import 'codemirror/addon/search/jump-to-line';
-import 'codemirror/addon/edit/matchbrackets';
-import 'codemirror/addon/edit/closebrackets';
-import 'codemirror/addon/selection/mark-selection';
-import 'codemirror/addon/hint/css-hint';
-import 'codemirror-colorpicker';
-
 import { JSHINT } from 'jshint';
 import { CSSLint } from 'csslint';
 import { HTMLHint } from 'htmlhint';
-import classNames from 'classnames';
-import { debounce } from 'lodash';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
-import MediaQuery from 'react-responsive';
-import '../../../../utils/htmlmixed';
-import '../../../../utils/p5-javascript';
-import { metaKey } from '../../../../utils/metaKey';
-import '../show-hint';
-import * as hinter from '../../../../utils/p5-hinter';
-import '../../../../utils/codemirror-search';
 
+import {
+  expandAbbreviation,
+  abbreviationTracker,
+  insert as emmetInsert,
+  resetAbbreviation
+} from '@emmetio/codemirror6-plugin';
+import {
+  findNext,
+  findPrevious,
+  highlightSelectionMatches
+} from '@codemirror/search';
+
+import classNames from 'classnames';
+import StackTrace from 'stacktrace-js';
```

Comment:
This doesn't necessarily have to be addressed in this PR, but wanted to note that [`stacktrace-js`](https://www.npmjs.com/package/stacktrace-js/v/0.6.1) is pretty outdated now should probably be replaced! Potentially with maybe [`strack-trace`](https://www.npmjs.com/package/stack-trace)?